### PR TITLE
[Feature] player, launcher manual update

### PR DESCRIPTION
**Pull request by Atralupus**
Solved https://github.com/planetarium/9c-launcher/issues/1908

Fix playerUpdate, launcherUpdate parameter, Add manual update buttons in settings overlay

### CODE REVIEW

**Review by Akamig — APPROVED** [review]

File: src/main/update/launcher-update.ts [hunk 1]
```diff
@@ -16,11 +15,6 @@ export async function launcherUpdate(
   update: IUpdate,
   listeners: IUpdateOptions
 ) {
-  if (!getConfig("UseUpdate", process.env.NODE_ENV === "production")) {
```
Comment (BasixKOR) [inline src/main/update/launcher-update.ts:19]: This got deleted, but it doesn't seem to be added anywhere else in this PR. Did we move this logic to other places before this?

**Review by BasixKOR — APPROVED** [review]
LGTM.

File: src/main/update/launcher-update.ts (hunk 1 above)
Comment (Atralupus) [reply src/main/update/launcher-update.ts:19]: yes, top of the `main.ts`(127 line)

Comment (BasixKOR) [reply src/main/update/launcher-update.ts:19]: Ooh I see. Thank you for checking in!

**Pull request merged**